src/Category.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+var Z = require('sanctuary-type-classes');
+
+var assert = require('./internal/assert');
+var compose = require('./internal/compose_');
+var id = require('./internal/id');
+
+
+module.exports = function(equals) {
+  return {
+
+    //  a `compose` id C = a
+    leftIdentity: assert.forall2(function(C, a) {
+      return Z.Category.test(a) &&
+             equals(compose(a)(id(C)),
+                    a);
+    }),
+
+    //  id C `compose` a = a
+    rightIdentity: assert.forall2(function(C, a) {
+      return Z.Category.test(a) &&
+             equals(compose(id(C))(a),
+                    a);
+    })
+
+  };
+};
```

Comment:
`a.compose(C.id())` is equivalent to `a` (right identity)
`C.id().compose(a)` is equivalent to `a` (left identity)

These look right to me. The identity element `C.id()` is on the right for "right identity" and on the left for "left identity".

The issue is `compose`. The arguments are swapped in the dispatch function (as they should be). See https://github.com/fantasyland/fantasy-land/pull/249